study.md

```diff
@@ -22,5 +22,21 @@ Ember](https://www.codeschool.com/courses/try-ember) course from CodeSchool.
 In your own words, describe why someone would use Ember.
 
 ```md
-<!-- your answer here -->
+Ember is a JavaScript MVC framework with a very established and detailed
+documentation. Ember works similarily to rails in that it makes small
+choices in the background for you to make programming more seamless. Ember
```

Comment:
The saying is that it favors "convention over configuration". Rails and Ember both follow this philosophy.